engine/NetworkProcessorNet.go

```diff
@@ -26,8 +26,8 @@ var _ = fmt.Print
 func NetworkProcessorNet(w *worker.Thread, fnode *fnode.FactomNode) {
 	FromPeerToPeer(w, fnode)
 	stubs(w, fnode)
-	BasicMessageValidation(w, fnode)
-	sort(w, fnode) // TODO: Replace this service entirely
+	BasicMessageValidation(w, fnode, 2) // 2 validators
```

Comment:
I'm not sure, I went with two because that's what the previous code used. I think it's likely to be premature optimization and just one will do fine. At some point, it will make sense to raise this (perhaps configurable), since this is basically the function that ingests messages from the p2p network and does things like signature checking.

You can ctrl+f for WellFormed() in the code to see that it's not yet entirely clear how heavy the load is.